waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -3,19 +3,26 @@
 import 
   options, chronos, stint,
   web3,
-  eth/keys
+  eth/keys,
+  libp2p/protobuf/minprotobuf,
+  stew/arrayops
 
 ## Bn256 and RLN are Nim wrappers for the data types used in 
 ## the rln library https://github.com/kilic/rln/blob/3bbec368a4adc68cd5f9bfae80b17e1bbb4ef373/src/ffi.rs
 type Bn256* = pointer
 type RLN*[E] = pointer
 
 
-type IDKey* = array[32, byte]
-type IDCommitment* = array[32, byte]
-# represents a Merkle tree node which is the output of 
-# Poseidon hash function implemented by rln lib
-type MerkleNode* = array[32,byte] 
+type 
+  IDKey* = array[32, byte]
```

Comment:
Can we add a brief comment above what these are or represent? Alternatively, add link to spec/docs where this is elaborated on.

Reply:
I have already explained these types in the `MembershipKeyPair` object definition (for each object field), will also include those comments here